utils/_context/_scenarios.py

```diff
@@ -721,7 +721,14 @@ class scenarios:
 
     appsec_waf_telemetry = EndToEndScenario(
         "APPSEC_WAF_TELEMETRY",
-        weblog_env={"DD_INSTRUMENTATION_TELEMETRY_ENABLED": "true", "DD_TELEMETRY_METRICS_INTERVAL_SECONDS": "2.0"},
+        weblog_env={
+            "DD_INSTRUMENTATION_TELEMETRY_ENABLED": "true",
+            "DD_TELEMETRY_METRICS_ENABLED": "true",
+            # Python lib has different env var until we enable Telemetry Metrics by default
+            "_DD_TELEMETRY_METRICS_ENABLED": "true",
+            "DD_TELEMETRY_METRICS_INTERVAL_SECONDS": "2.0",
+            "DD_APPSEC_RULES": None,
```

Comment:
why this one ?

Reply:
Originally to avoid inhibiting blocking, but it's actually not needed because we're not using IP blocking here. Removed.